plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/tools/PostgreBackupAllWizardPageSettings.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.postgresql.tools;
+
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.*;
+import org.jkiss.dbeaver.ext.postgresql.PostgreMessages;
+import org.jkiss.dbeaver.ext.postgresql.tasks.PostgreBackupAllSettings;
+import org.jkiss.dbeaver.ui.UIUtils;
+import org.jkiss.utils.CommonUtils;
+
+public class PostgreBackupAllWizardPageSettings extends PostgreToolWizardPageSettings<PostgreBackupAllWizard> {
+
+    private Combo encodingCombo;
+    private Button metadataOnly;
+    private Button globalsOnly;
+    private Button rolesOnly;
+    private Button tablespacesOnly;
+    private Button noPrivilegesCheck;
+    private Button noOwnerCheck;
+    private Button addRolesPasswords;
+
+    public PostgreBackupAllWizardPageSettings(PostgreBackupAllWizard wizard) {
+        super(wizard, PostgreMessages.wizard_backup_all_page_setting_title_setting);
+        setTitle(PostgreMessages.wizard_backup_all_page_setting_title);
+        setDescription(PostgreMessages.wizard_backup_all_page_setting_title);
+    }
+
+    @Override
+    public void createControl(Composite parent) {
+        Composite composite = UIUtils.createPlaceholder(parent, 1);
+        SelectionListener changeListener = new SelectionAdapter() {
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                updateState();
+            }
+        };
```

Comment:
Let's try adapting Eclipse factories for creating adapters:

```suggestion
        SelectionListener changeListener = SelectionListener.widgetSelectedAdapter(e -> updateState());
```